README.md

```diff
@@ -106,6 +106,52 @@ buildctl build ... --exporter=local --exporter-opt output=path/to/output-dir
 buildctl du -v
 ```
 
+#### Running containerized buildkit
+
+buildkit daemon supports communicating with clients over TCP. This is supports some use cases like:
+
+* Docker for Mac or Windows users which cannot run natively buildkit daemon on their platform,
+* Centralized daemon for remote build scripts execution in a team settings
+* ...
+
+To run daemon in a container:
+
+```
+$ docker run --rm --privileged -p 12345:12345 --tmpfs /tmp  buildkit:buildd-standalone buildd-standalone --addr tcp://0.0.0.0:12345 --root /tmp/buildkit
+```
+
+Notes:
+
+* `--privileged` is needed for buildkit to access docker engine
```

Comment:
`buildkit` doesn't access docker engine. `--privileged` is needed to grant permissions for assigning namespaces and allow mounting snapshots.